### ISSUE TITLE
G Drive Space full

### ISSUE THREAD

**Issue by gotschi** [sabeechen/hassio-google-drive-backup#341]
###### Description:
```
The addon tells me that my G drive space is full, but it's only 1.8Gb used of 15Gb,
is there sth wrong with my setup?

thanks for this great addon!
```

 Addon version: `0.103.1`
 Home Assistant Version: `2021.2.1`
 Supervisor Version: `2021.01.7`
 Supervisor Channel: `stable`
 Hassos Version: `None`
 Docker Version: `19.03.13`
 Architecture: `aarch64`
 Machine: `raspberrypi4-64`
 Date: `2021-02-06T08:24:23.268894+00:00`
 Timezone: `Europe/Vienna`
 Failure Time: `2021-02-06T08:16:46.755459+00:00`
 Last Good Sync: `2021-02-06T08:16:46.173017+00:00`
 ###### Exception:
 ```
 
  addon/backup/model/coordinator.py:156 (_sync)
  addon/backup/model/model.py:180 (sync)
  addon/backup/drive/drivesource.py:160 (save)
  addon/backup/drive/driverequests.py:232 (create)
  addon/backup/drive/driverequests.py:330 (retryRequest)
  addon/backup/creds/driverequester.py:32 (request)
  addon/backup/creds/driverequester.py:105 (raiseForKnownErrors)
backup.exceptions.exceptions.DriveQuotaExceeded

 ```
 Snapshots:
 ```
 {
    "HomeAssistant": {
        "snapshots": 5,
        "retained": 0,
        "deletable": 5,
        "name": "HomeAssistant",
        "title": "Home Assistant",
        "latest": "2021-02-06T08:16:44Z",
        "max": 4,
        "enabled": true,
        "size": "323.2 GB",
        "free_space": "431.5 GB"
    },
    "GoogleDrive": {
        "snapshots": 4,
        "retained": 0,
        "deletable": 4,
        "name": "GoogleDrive",
        "title": "Google Drive",
        "latest": "2021-02-06T02:00:05Z",
        "max": 4,
        "enabled": true,
        "size": "169.7 MB"
    }
}
 ```
 ###### Config:
 ```
 {
    "Setting.DAYS_BETWEEN_SNAPSHOTS": 1.0,
    "Setting.SNAPSHOT_TIME_OF_DAY": "03:30",
    "Setting.SNAPSHOT_PASSWORD": "REDACTED",
    "Setting.EXCLUDE_FOLDERS": "share"
}
 ```
 ###### Addon Logs:
 ```
 02-06 09:01:46 ERROR [backup.model.coordinator] Google Drive is out of space
02-06 09:01:46 INFO [backup.model.coordinator] I'll try again in an hour
02-06 09:16:44 DEBUG [backup.model.syncer] Sync requested by HomeAssistant
02-06 09:16:44 INFO [backup.model.coordinator] Syncing Snapshots
02-06 09:16:44 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/snapshots
02-06 09:16:44 DEBUG [backup.drive.driverequests] Requesting refreshed Google Drive credentials
02-06 09:16:45 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/1J3RoGZIf3eGXkn3-k2YHNgLaLE0OMuCg/?fields=id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents&supportsAllDrives=true
02-06 09:16:45 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%271J3RoGZIf3eGXkn3-k2YHNgLaLE0OMuCg%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
02-06 09:16:45 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/addons/self/info
02-06 09:16:45 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/info
02-06 09:16:45 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/core/info
02-06 09:16:45 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/supervisor/info
02-06 09:16:45 INFO [backup.ha.hasource] Requesting a new snapshot
02-06 09:16:45 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/snapshots/new/partial
02-06 09:16:45 WARNING [backup.ha.hasource] A snapshot was already in progress
02-06 09:16:45 ERROR [backup.model.coordinator] A snapshot is already in progress
02-06 09:16:45 INFO [backup.model.coordinator] I'll try again in an hour
02-06 09:16:46 DEBUG [backup.model.syncer] Sync requested by HomeAssistant
02-06 09:16:46 INFO [backup.model.coordinator] Syncing Snapshots
02-06 09:16:46 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/snapshots
02-06 09:16:46 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%271J3RoGZIf3eGXkn3-k2YHNgLaLE0OMuCg%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
02-06 09:16:46 INFO [backup.drive.drivesource] Uploading 'Partial Snapshot 2021-02-05 03:30:00' to Google Drive
02-06 09:16:46 DEBUG [backup.drive.driverequests] Starting a new upload session with Google Drive
02-06 09:16:46 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true
02-06 09:16:46 ERROR [backup.model.coordinator] Google Drive is out of space
02-06 09:16:46 INFO [backup.model.coordinator] I'll try again in an hour
02-06 09:23:59 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/addons/self/info
02-06 09:23:59 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/info
02-06 09:23:59 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/core/info
02-06 09:23:59 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/supervisor/info
 ```
 ###### Supervisor Logs:
 ```
 [32m21-02-06 05:12:52 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m21-02-06 05:42:59 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m21-02-06 06:01:52 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 06:01:53 INFO (MainThread) [supervisor.api.security] /snapshots/3a828cf5/download access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 06:01:53 INFO (MainThread) [supervisor.api.snapshots] Downloading snapshot 3a828cf5[0m
[32m21-02-06 06:13:06 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m21-02-06 06:43:15 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m21-02-06 07:01:53 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 07:01:54 INFO (MainThread) [supervisor.api.security] /snapshots/3a828cf5/download access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 07:01:54 INFO (MainThread) [supervisor.api.snapshots] Downloading snapshot 3a828cf5[0m
[32m21-02-06 07:13:22 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m21-02-06 07:43:30 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m21-02-06 08:01:44 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:01:45 INFO (MainThread) [supervisor.api.security] /snapshots/3a828cf5/download access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:01:45 INFO (MainThread) [supervisor.api.snapshots] Downloading snapshot 3a828cf5[0m
[32m21-02-06 08:13:37 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m21-02-06 08:16:44 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:16:45 INFO (MainThread) [supervisor.api.security] /core/info access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:16:45 INFO (MainThread) [supervisor.api.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:16:45 INFO (MainThread) [supervisor.api.security] /snapshots/new/partial access from cebe7a76_hassio_google_drive_backup[0m
[33m21-02-06 08:16:45 WARNING (MainThread) [supervisor.jobs] 'SnapshotManager.do_snapshot_partial' blocked from execution, system is not running[0m
[32m21-02-06 08:16:46 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:16:46 INFO (MainThread) [supervisor.api.security] /snapshots/3a828cf5/download access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:16:46 INFO (MainThread) [supervisor.api.snapshots] Downloading snapshot 3a828cf5[0m
[32m21-02-06 08:22:56 INFO (MainThread) [supervisor.api.security] /supervisor/logs access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:22:56 INFO (MainThread) [supervisor.api.security] /core/logs access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:23:59 INFO (MainThread) [supervisor.api.security] /core/info access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:23:59 INFO (MainThread) [supervisor.api.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup[0m
[32m21-02-06 08:24:23 INFO (MainThread) [supervisor.api.security] /supervisor/logs access from cebe7a76_hassio_google_drive_backup[0m


 ```

**Comment by sabeechen** [comment]
Are you using the space in any other products, like GMail or Google Photos?  They all share the same 15GB, and you should be able to see the usage of each if you go to [https://one.google.com/storage](https://one.google.com/storage)

**Comment by gotschi** [comment]
yes I took a look online - that's where it said I used 1.8gb

but the issue seems solved since the upgrade to 2021.2.2,
will close :)

thanks
